tests/model_registry/rbac/conftest.py

```diff
@@ -89,6 +93,38 @@ def sa_token(service_account: ServiceAccount) -> str:
         raise
 
 
+@pytest.fixture(scope="session", autouse=True)
+def check_env_vars() -> None:
+    """
+    Check if the required environment variables are set.
+    Fails the test session if any are missing.
+    """
+    missing: list[str] = []
+    missing.extend(var for var in ["ADMIN_PASSWORD", "NON_ADMIN_PASSWORD"] if not os.environ.get(var))
+    if missing:
+        pytest.skip(f"Required environment variables not set: {', '.join(missing)}")
+
+
+@pytest.fixture(scope="function")
+def new_group(request: pytest.FixtureRequest) -> Generator[str, None, None]:
+    """
+    Fixture to create a new OpenShift group and add a user, then delete the group after the test.
+    The parameter should be a tuple: (group_name, user_name).
+    """
+    group_name, user_name = request.param
+    try:
+        run_command(command=["oc", "adm", "groups", "new", group_name, user_name])
+    except Exception as e:
+        pytest.skip(f"Failed to create group {group_name} and add user {user_name}: {e}")
```

Comment:
We should raise it. This can potentially cause silent skippping of tests and we don't want that.